.github/workflows/matrix.yml

```diff
@@ -161,18 +158,18 @@ jobs:
             ccache_limit: 6G
             ccache_key: linux-llvm-18-asan
 
-          - compiler: g++-12
-            os: ubuntu-22.04
+          - compiler: g++-14
+            os: ubuntu-latest
             cmake: 0
             tiles: 0
             sound: 0
             localize: 1
             native: linux64
             pch: 1
-            sanitize: address
-            title: GCC 12, Ubuntu, Curses, ASan
+          # sanitize: address
```

Comment:
If you're going to remove it remove it, plus this'll rebase the branch on latest master.
```suggestion
```